router.go

```diff
@@ -6,9 +6,10 @@ package httptreemux
 
 import (
 	"fmt"
-	"github.com/dimfeld/httppath"
 	"net/http"
 	"net/url"
+
+	"github.com/dimfeld/httppath"
```

Comment:
Can you run the file through `gofmt`? I think that will place this line back in its original location.